packages/mobile/src/firebase/firebase.test.ts

```diff
@@ -87,13 +87,13 @@ describe(initializeCloudMessaging, () => {
 
 describe('Firebase version check', () => {
   it('Correctly check if version is deprecated', () => {
-    let isDeprecated: boolean = isDeprecatedVersion('1.5.0', '1.4.0')
+    let isDeprecated: boolean = isVersionBelowMinimum('1.5.0', '1.4.0')
```

Comment:
maybe you don't need to re-set that variable here and just 
`expect(isVersionBelowMinimum('1.5.0', '1.4.0')).toBe(false)`

Reply:
Fixed